ui/app/AppLayouts/Onboarding/views/ConfirmPasswordView.qml

```diff
@@ -106,24 +101,34 @@ Page {
             id: submitBtn
             anchors.horizontalCenter: parent.horizontalCenter
             text: qsTr("Finalize Status Password Creation")
-            enabled: !submitBtn.loading && confPswInput.text === root.password
+            enabled:!submitBtn.loading && confPswInput.text === root.password
 
             property Timer sim: Timer {
                 id: pause
                 interval: 20
                 onTriggered: {
-                    // Create new password call action to the backend
-                    OnboardingStore.onBoardingModul.storeSelectedAccountAndLogin(root.password)
-                    Global.applicationWindow.prepareForStoring(root.password, false)
+                    // Create account operation blocks the UI so loading = true; will never have any affect until it is done.
+                    // Getting around it with a small pause (timer) in order to get the desired behavior
+                    OnboardingStore.finishCreatingAccount(root.password)
                 }
             }
 
             onClicked: {
-                confPswInput.text = ""
-                submitBtn.loading = true
-                // Create password operation blocks the UI so loading = true; will never have any affect until changePassword/createPassword is done.
-                // Getting around it with a small pause (timer) in order to get the desired behavior
-                pause.start()
+                //confPswInput.text = ""
+                if (OnboardingStore.accountCreated) {
+                    if (root.password !== root.tmpPass) {
+                        OnboardingStore.changePassword(root.tmpPass, root.password);
```

Comment:
@alexandraB99 could you clarify me which is the flow for arriving here? I'm now confused about why to use `changePassword` in the onboarding flow if it is suposed to be a new account or an existing one but by entering the seed phrase (so you should have to create a new password as well)

Reply:
yeah, so in case the account is created, the biometrics screen is shown but the user decides to go back and change things.

Reply:
Ok!! I understand now!! thanks :)